src/core/ext/filters/client_channel/subchannel.cc

```diff
@@ -58,12 +58,12 @@
 #define INTERNAL_REF_BITS 16
 #define STRONG_REF_MASK (~(gpr_atm)((1 << INTERNAL_REF_BITS) - 1))
 
-// Backoff parameters.
-#define GRPC_SUBCHANNEL_INITIAL_CONNECT_BACKOFF_SECONDS 1
```

Comment:
We should keep these names consistent with the ones in the backoff doc:

https://github.com/grpc/grpc/blob/master/doc/connection-backoff.md

Otherwise, it's gets very confusing as to which ones are which when trying to verify if our behavior matches the documented behavior.  (There are a lot of subtleties here; we had to go through several iterations of the backoff code when we converted it to C++.)

Reply:
Using the full name in the doc might be a little too verbose. I added the arg names for the configurable parameters.

Reply:
To be clear, I was not suggesting changing the doc; I was suggesting that the names here in the code should more closely match the names in the doc.  I suggest the following renames from what you have here now:

- DEFAULT_INITIAL_RECONNECT_BACKOFF_MS -> DEFAULT_RECONNECT_INITIAL_BACKOFF_MS
- DEFAULT_MAX_RECONNECT_BACKOFF_MS -> DEFAULT_RECONNECT_MAX_BACKOFF_MS
- DEFAULT_MIN_CONNECT_TIMEOUT_MS -> DEFAULT_MIN_CONNECT_TIMEOUT

Note that the doc is not C-core-specific; it's describing an algorithm that is used by all grpc implementations.  As such, we probably don't want C-core-specific documentation in it.